app/jobs/deposit_work_job.rb

```diff
@@ -49,4 +51,14 @@ def perform_persist
   def assign_doi
     work_form.doi_option == 'yes'
   end
+
+  def update_terms_of_deposit
```

Comment:
Slight preference to see this as a bang method to surface that it's a mutator but not a blocking concern.

Reply:
Concur. Fixed.